Tests/PSSwaggerScenario.Tests.ps1

```diff
@@ -846,3 +846,218 @@ Describe "Header scenario tests" -Tag @('Header','ScenarioTest')  {
         }
     }
 }
+
+Describe "Pre-compiled SDK Assmebly scenario tests" -Tag @('SDKAssembly','ScenarioTest')  {    
+    # As of today 9/7/2017, PSSwagger is not supported on PowerShell Core.
+    if((Get-Variable -Name PSEdition -ErrorAction Ignore) -and ('Core' -eq $PSEdition)) {
+        return
+    }
```

Comment:
Ideally: generalize the helper function that takes care of core vs. full. No reason why these tests shouldn't run for core as well.

Reply:
ee36770